tst/MetricsApiTest.cpp

```diff
@@ -156,6 +156,71 @@ TEST_F(MetricsApiTest, webRtcIceCandidateGetMetrics)
     EXPECT_EQ(STATUS_SUCCESS, freePeerConnection(&pRtcPeerConnection));
 }
 
+TEST_F(MetricsApiTest, webRtcTransportGetMetrics)
+{
+    RtcConfiguration configuration;
+    PRtcPeerConnection pRtcPeerConnection = NULL;
+    RtcStats rtcTransportMetrics;
+
+    MEMSET(&configuration, 0x00, SIZEOF(RtcConfiguration));
+    STRNCPY(configuration.iceServers[0].urls, (PCHAR) "stun:stun.kinesisvideo.us-west-2.amazonaws.com:443", MAX_ICE_CONFIG_URI_LEN);
+    STRNCPY(configuration.iceServers[0].credential, (PCHAR) "", MAX_ICE_CONFIG_CREDENTIAL_LEN);
```

Comment:
I saw this macro that we can use instead:
https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/65e38dac9b30523d43a57bc009d679e627b58d9a/src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h#L878

Reply:
Done